chapters/intro-computer-architecture/c-goto-statements/guides/goto-warm-up/README.md

```diff
@@ -0,0 +1,22 @@
+# C: Warm-up GOTOs
+
+**1.** Modify the source code in the
+`support/bogosort/bogosort.c`
+file, by replacing the `break` instruction with a `goto` instruction.
+([Bogosort](https://en.wikipedia.org/wiki/Bogosort))
+
+**2.** Similarly, replace the `continue` instruction in
+`support/ignore_the_comments/ignore_the_comments.c`
+with a `goto` instruction without changing the functionality of the code.
+
+>**WARNING**: When writing code with labels, please adhere to the following indentation recommendations:
```

Comment:
As I said in another comment on another suggestion, the text si broken up onto multiple lines because the linter would say there should only be one sentence per line. I could try to commit the suggestion and see if I can find a better fix.